bin/broadside

```diff
@@ -44,9 +46,9 @@ end
 
 desc 'Bootstrap your service and task definition from the configured definition.'
 command :bootstrap do |b|
-  subcmd.desc 'Docker tag for application container'
-  subcmd.arg_name 'TAG'
-  subcmd.flag [:tag]
+  b.desc 'Docker tag for application container'
+  b.arg_name 'TAG'
+  b.flag [:tag]
```

Comment:
also i'm not sure how this ever worked before this change... there is no `subcmd` in scope here?  /shrug